protocol_driver_grpc_async_callback.cc

```diff
@@ -104,6 +104,7 @@ void ProtocolDriverGrpcAsyncCallback::SetNumPeers(int num_peers) {
 }
 
 ProtocolDriverGrpcAsyncCallback::~ProtocolDriverGrpcAsyncCallback() {
+  ShutdownClient();
   ShutdownServer();
   grpc_client_stubs_.clear();
```

Comment:
this line of code should be inside shutdown client, and I think we should be calling shutdown server before shutdown client.

Reply:
Done